apps/google-analytics-4/lambda/serverless.yml

```diff
@@ -1,26 +1,105 @@
 service: google-analytics
-frameworkVersion: '3'
+frameworkVersion: '3' #might need to switch to '>=2.0.0 <3.0.0'
+
+variablesResolutionMode: 20210326
 
 plugins:
+  - serverless-domain-manager
   - serverless-offline
 
 custom:
-  environment: ${file(./config/serverless-env.${opt:stage, 'dev'}.yml)}
+  signingSecret:
+    prd: ${ssm:/aws/reference/secretsmanager/ci/apps/ga4-build-action/prd/signing-secret} #assuming these will exist
+    test: ${ssm:/aws/reference/secretsmanager/ci/apps/ga4-build-action/test/signing-secret}
+  appIdentityKey:
+    prd: ${ssm:/aws/reference/secretsmanager/ci/apps/ga4-build-action/prd/app-identity-key}
+    test: ${ssm:/aws/reference/secretsmanager/ci/apps/ga4-build-action/test/app-identity-key}
+  domainName:
+    prd: ga4-build.ctfapps.net
+    test: ga4-build-test.ctfapps.net
+  appDefinitionId:
+    prd: 5eHTUt9pILTGjYk3VFE9ta
```

Comment:
This is not the productionalized id, will have to change.

(How would we know it before deploying this?)

Reply:
@jsdalton mentioned in standup we have the production app definition ID created now. So this can be updated to: `5DlxOS0KvGS1Wk362xgvbN`

Reply:
already done!